Source/SelfService/Web/microservice/microserviceDetails/configurationFilesSection/setupSection/setupSection.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { useSnackbar } from 'notistack';
+
+import { Box } from '@mui/material';
+
+import { Accordion, ConfirmDialog, Form } from '@dolittle/design-system';
+
+import { canDeleteMicroservice, deleteMicroservice, MicroserviceStore } from '../../../../stores/microservice';
+
+import { HttpResponseApplication } from '../../../../api/application';
+import { MicroserviceFormParameters } from '../../../../api/index';
+
+import { HasM3ConnectorField } from '../../../components/form/hasM3ConnectorField';
+import { RestartMicroserviceDialog } from '../../../components/restartMicroserviceDialog';
+import { HeaderButtons } from './headerButtons';
+import { SetupFields } from './setupFields';
+import { ContainerImageFields } from './containerImageFields';
+import { PublicUrlFields } from './publicUrlFields';
+import { getRuntimeNumberFromString } from '../../../helpers';
+
+const styles = {
+    form: {
+        'mt': 3,
+        '& .MuiFormControl-root': {
+            'my': 1,
+            '& fieldset': {
```

Comment:
the `dashed` part here seems like something that could be part of another component? Unless its only used here?

Reply:
It is only used here at the moment. I think that 'dashed input' need to be added into Design System as Input variant or so...